dist/api_data.py

```diff
@@ -1582,6 +1583,14 @@ def __ge__(self, other):
         where the application is rolling back locally committed transactions.
         The supplied value must not be older than the current oldest and
         stable timestamps.  See @ref transaction_timestamps'''),
+    Config('durable_timestamp', '', r'''
```

Comment:
I know the plan is to remove commit_timestamp from here - but should we mark it as deprecated and update the documentation to say that it's a synonym for durable_timestamp, in case we don't end up doing that soon?

Reply:
3e0eab0

Marked as deprecated in docs.